salt/utils/functools.py

```diff
@@ -45,7 +46,7 @@ def alias_function(fun, name, doc=None):
     '''
     alias_fun = types.FunctionType(fun.__code__,
                                    fun.__globals__,
-                                   str(name),  # future lint: disable=blacklisted-function
+                                   salt.utils.stringutils.to_str(name),  # future lint: disable=blacklisted-function
```

Comment:
`str()` is actually what we want here, `and in fact salt.utils.stringutils.to_str()` will raise an exception if you feed it value that isn't one of the string types.